opa/rego/rules/injection.rego

```diff
@@ -77,3 +77,27 @@ results contains poutine.finding(rule, pkg.purl, {
 	exprs := gl_injections(script)
 	count(exprs) > 0
 }
+
+# Azure Pipelines
+patterns.azure contains `\$\((Build\.(SourceBranchName|SourceBranch|SourceVersionMessage)|System\.PullRequest\.SourceBranch)\)`
```

Comment:
Practically speaking it looks like only `System.PullRequest.SourceBranch` is truly user controllable, but the fact remains that all the others fit the definition, just that in all my test scenarios so far they are referirng to merge commit and the merge commit message.

That being said, the other may or may not be exploitable when the PR gets merged though... likely , I'll need to test, but good let's keep the regex like this